lib/textParsers.js

```diff
@@ -234,6 +234,47 @@ var parseJsonArray = function(val) {
   return arr.map(function(el) { return JSON.parse(el); });
 };
 
+var parsePoint = function(val) {
+  if (val[0] !== '(') { return null; }
+
+  val = val.substring( 1, val.length - 1 ).split(',');
+
+  return {
+    x: parseFloat(val[0])
+  , y: parseFloat(val[1])
+  };
+};
+
+var parseCircle = function(val) {
```

Comment:
I opted to avoid regex because the point could be a set of integers or decimals and figured a token-based `O(n)` algorithm would just be better.